### ISSUE TITLE
an error withleak_tracker_testing-2.0.3 after update flutter to 3.19.2 

### ISSUE THREAD

**Issue by AhkamKhalaaf** [dart-lang/leak_tracker#222]
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/leak_tracker_testing-2.0.3/lib/src/leak_testing.dart:146:9: Error: No named
parameter with the name 'notGCed', 
how to fix it plz

**Comment by azozaod5** [comment]
did you fix it? 
if you, how plz?

**Comment by adnankovacevic** [comment]
Upgrade Flutter to 3.19.3 version

**Comment by AhkamKhalaaf** [comment]
> did you fix it? if you, how plz?

i use 3.19.0 and it fixed

**Comment by rontho** [comment]
Can someone have a look at this ? I'm facing the same issue and upgrading to 3.19.3 did not fix the issue. I'm using 3.19.2. 

Thank you

**Comment by Mudassar-Afzal-Ch** [comment]
@rontho any luck?

**Comment by rontho** [comment]
> @rontho any luck?

No, I'm still facing the same issue. I've tried to specified a newer version of the leak_tracker in my project but I get that issue : _Note: leak_tracker_testing is pinned to version 2.0.3 by flutter_test from the flutter SDK._

I also tried to remove any dependencies from the testing library and comment all the test in the project but it seems that this library is part of the flutter framework. 

I tried to remove pubspec.lock, clear Android studio cache, clear project cache, repair cache... nothing worked.

**Comment by rontho** [comment]
Ok I manage to fix the issue. I was using AndroidStudio, and when I tried to create a new Flutter project from scratch I had the same error in Android Studio editor "leak_tracker_testing is pinned to version 2.0.3 by flutter_test from the flutter SD"

So I realized the problem was not in my project but in my flutter config I had installed on my machine. 

I got ride of the flutter I had installed using homebrew, then follow the instruction on flutter.dev to install flutter using VSCODE. 

And after cleaning the cache again, everything was working fine.

Hope this will help

**Comment by wagnerjk** [comment]
I managed to solve it by adding to pubspec.yaml: 

```
dependency_overrides:
  leak_tracker_flutter_testing: 3.0.3
  leak_tracker_testing: 3.0.1
```

**Comment by ishutinvv** [comment]
In my case `flutter` directory was dirty. So, cleaning it up solved the issue:
`pushd "$(which flutter)/../../" && git checkout . && popd`

**Comment by simplekjl** [comment]
@wagnerjk thanks! the solution works, in this case I guess this is a short term solution, which one would be the best approach?

**Comment by westy92** [comment]
@ishutinvv my `flutter` directory was dirty too, but your suggestion didn't work.

I had to:
* reinstall flutter
* clear the cache: `dart pub cache clean`
* run: `flutter pub get`
